docs/manual/developer/04_style_guide.md

```diff
@@ -52,6 +52,62 @@ Noteworthy additions to these principles:
 * Should use the [merge commit method](https://docs.github.com/en/github/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges)
 * Should use the GitHub Web UI to document and ensure code reviews are done correctly
 
+### Commit Messages
+
```

Comment:
I would mention this is based on the post you linked.

Reply:
Addressed